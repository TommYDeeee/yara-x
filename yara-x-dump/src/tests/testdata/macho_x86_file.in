field1: 123
field2: "test"
field3: "test\ntest"
segments {
    nested1: 456
    nested2: 789
    timestamp: 123456789
}
segments {
    nested1: 100000
    nested2: 200000
    timestamp: 999999999
}
optional {
    onested1: 123
    onested2: 456
    map_string_string {
      key: "foo\nfoo"
      value: "bar\nbar"
    }
}
