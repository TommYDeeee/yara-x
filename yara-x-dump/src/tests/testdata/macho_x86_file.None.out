[38;2;51;51;255mfield1[0m: 123 (0x7b)
[38;2;51;51;255mfield2[0m: "test"
[3;38;2;51;255;153m# Nested[0m [3;38;2;51;255;153msegments[0m [3;38;2;51;255;153mstructure[0m
[1;38;2;255;255;102msegments[0m:
  [1;38;2;255;255;102m-[0m [38;2;51;51;255mnested1[0m: 456
    [38;2;51;51;255mnested2[0m: 789 (0x315)
    [38;2;51;51;255mtimestamp[0m: 123456789 (1973-11-29 21:33:09 UTC)
[3;38;2;51;255;153m# Nested[0m [3;38;2;51;255;153moptional[0m [3;38;2;51;255;153mstructure[0m
[1;38;2;255;255;102moptional[0m:
  [1;38;2;255;255;102m-[0m [38;2;51;51;255monested1[0m: 123
    [38;2;51;51;255monested2[0m: 456 (0x1c8)
    [1;38;2;255;255;102mmap_string_string[0m:
        [38;2;51;51;255mfoo[0m: "bar"
